apps/astarte_trigger_engine/lib/astarte_trigger_engine/events_consumer.ex

```diff
@@ -296,9 +320,9 @@ defmodule Astarte.TriggerEngine.EventsConsumer do
     with {:ok, client} <- Database.connect(realm: realm_name),
          {:ok, result} <- DatabaseQuery.call(client, query),
          [value: trigger_data] <- DatabaseResult.head(result),
-         trigger <- Trigger.decode(trigger_data),
+         %Trigger{} = trigger <- Trigger.decode(trigger_data),
          {:ok, action} <- Jason.decode(trigger.action) do
-      {:ok, action}
+      {:ok, trigger.name, action}
```

Comment:
What if this returned `{:ok, trigger}`? By passing the trigger around you would reduce the number of parameters in function calls (e.g. `event_to_payload`) making it easier to read the code